Orange/widgets/data/owpreprocess.py

```diff
@@ -637,55 +637,82 @@ def enum_to_index(enum, key):
     enum key to its int position"""
     return list(enum).index(key)
 
-
 class Scale(BaseEditor):
-    NoCentering, CenterMean, CenterMedian = 0, 1, 2
-    NoScaling, ScaleBySD, ScaleBySpan = 0, 1, 2
+    NoCentering, CenterByMean, CenterByMedian, NormalizeSD, NormalizeSpan, NormalizeSpan2 = 0, 1, 2, 3, 4, 5
```

Comment:
`NormalizeSpan2` is not very descriptive. :) What about `NormalizeBySD`, `NormalizeBySpan_ZeroBased`, `NormalizeSpan_NonZeroBased`. It is not entirely clear what the latter is based on, but it's still closer to being understandable. :)